RallySimulator.Persistence/RallySimulator.Persistence.csproj

```diff
@@ -1,14 +1,15 @@
 <Project Sdk="Microsoft.NET.Sdk">
 
   <PropertyGroup>
-    <TargetFramework>netstandard2.1</TargetFramework>
+    <TargetFramework>net6.0</TargetFramework>
+	<AnalysisMode>AllEnabledByDefault</AnalysisMode>
```

Comment:
Let's make the indent the same as it was before - I think it's 2 spaces

Reply:
They are already done, ı have calculated spaces. Just 2 spaces.

Reply:
Weird, but take a look at the diff on Github. You will see what I mean
![image](https://user-images.githubusercontent.com/34191235/188300892-a4105c2d-ac81-4a47-a002-752577039c8a.png)

Reply:
![image](https://user-images.githubusercontent.com/53901858/188303596-9f0e92bc-4175-4131-a337-51ebaf9857c3.png)

You're right it seems different on GitHub...